lib/game_eval.rb

```diff
@@ -2,7 +2,7 @@ module TicTacToe
 
   class GameEval
 
-    def player_won(board, marker1, marker2)
+    def player_marker_won(board, marker1, marker2)
```

Comment:
This method name still sounds like it may be returning a boolean value. What do you think about `#winning_player_marker` or `#winning_marker`?